controllers/patientController.js

```diff
@@ -236,6 +269,7 @@ const dischargePatientById = async (req, res) => {
     }
 
     patient.isDischarged = true;
+    patient.dischargeDateTime = new Date();
```

Comment:
this will be in UTC+0 when creating so gotta offset it by 8hrs to get SGT

![telegram-cloud-photo-size-5-6219799324435528814-y](https://github.com/AHVirtualNursing/virtual-nursing-server/assets/77633638/341d3681-fb07-46b7-961a-aaca3ecd43c5)